fearless/Common/Extension/UIKit/UIButton.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+import UIKit
+
+extension UIButton {
+    func textAndImage(spacing: CGFloat) {
```

Comment:
set keyword missed. let's name it 'setTextAndImageSpacing(_ spacing: CGFloat)' or 'setContentSpacing(_ spacing: CGFloat)'